guides/common/assembly_understanding-multiple-content-view-environments.adoc

```diff
@@ -0,0 +1,23 @@
+include::modules/con_understanding-multiple-content-view-environments.adoc[]
+
+include::modules/con_content-view-environment-categories.adoc[leveloffset=+1]
+
+include::modules/proc_assigning-multiple-content-view-environments-to-hosts.adoc[leveloffset=+1]
+
+include::modules/proc_assigning-multiple-content-view-environments-to-hosts-with-subscription-manager.adoc[leveloffset=+1]
+
+include::modules/proc_viewing-multiple-content-view-environment-hosts.adoc[leveloffset=+1]
+
+include::modules/proc_assigning-multiple-content-view-environments-to-activation-keys.adoc[leveloffset=+1]
```

Comment:
I think the AK modules would make more sense on the Managing AKs assembly and linked from this assembly. Alternatively, you could change the offset of the other AK modules to +2, so that they appear under this module. 

With so many leveloffset=+1 headers, it's not easy to grasp the overall structure of this assembly. I think a user might find it difficult to figure out the order of the steps when faced with so many choices.

Reply:
I'd be +1 to increasing the level.